govtool/backend/sql/get-network-metrics.sql

```diff
@@ -36,11 +77,70 @@ total_drep_votes AS (
     WHERE
         voter_role = 'DRep'
 ),
+total_stake_controlled_by_dreps AS (
+    SELECT
+        SUM(dd.amount)::bigint AS total
+    FROM
+        drep_distr dd
+),
+total_registered_direct_voters AS (
+    SELECT 
+        COUNT(DISTINCT dh.raw) AS unique_direct_voters
+    FROM 
+        drep_registration dr
+    JOIN
+        drep_hash dh
+    ON
+        dr.drep_hash_id = dh.id
+    LEFT JOIN 
+        voting_anchor va
+    ON 
+        dr.voting_anchor_id = va.id
+    WHERE 
+        dr.deposit > 0 
+        AND va.url IS NULL
+),
 total_registered_dreps AS (
+    SELECT 
+        count(DISTINCT dh.raw) AS unique_registrations
+    FROM 
+        drep_registration dr
+    JOIN
+        drep_hash dh
+    ON
+        dr.drep_hash_id = dh.id
+    WHERE 
+        dr.deposit > 0
+),
+total_active_dreps AS (
     SELECT
-        count(*) AS count
+        count(DISTINCT drep_hash_raw) AS unique_active_drep_registrations
+    FROM
+        RankedDRep
+    WHERE
+        epoch_no >= (SELECT epoch_no FROM active_drep_boundary_epoch)
+        AND rank = 1
+),
+total_inactive_dreps AS (
+    SELECT
+        total_registered_dreps.unique_registrations - total_active_dreps.unique_active_drep_registrations AS total_inactive_dreps
     FROM
-        drep_hash
+        total_registered_dreps
+    CROSS JOIN
+        total_active_dreps
+),
+total_active_cip119_compliant_dreps AS (
+    SELECT
+        count(DISTINCT drep_hash_raw) AS unique_active_cip119_compliant_drep_registrations
+    FROM
+        RankedDRep
+    JOIN
+        voting_anchor va on va.id = RankedDRep.voting_anchor_id
+    JOIN off_chain_vote_data ocvd on ocvd.voting_anchor_id = va.id
+    JOIN off_chain_vote_drep_data ocvdd on ocvdd.off_chain_vote_data_id = ocvd.id
+    WHERE
+    -- given_name is the only compulsory field in CIP-119
+        ocvdd.given_name IS NOT NULL
```

Comment:
Shouldn't we also check if the drep is actually active?

```suggestion
        ocvdd.given_name IS NOT NULL
        AND epoch_no >= (SELECT epoch_no FROM active_drep_boundary_epoch)
        AND rank = 1
```